src/odemis/acq/move.py

```diff
@@ -2262,3 +2755,164 @@ def _getCurrentAlignerPositionLabel(self) -> int:
             return LOADING_PATH
         # None of the above -> unknown position
         return UNKNOWN
+
+class SampleStage(model.Actuator):
+    """
+    Stage wrapper component which converts the stage position to the sample stage position.
+    The sample stage coordinates system is along the sample-plane which is adjusted
+    according to the pre-tilt and other factors.
+    """
+
+    def __init__(self, name: str, role: str, stage_bare: model.Actuator , posture_manager: MicroscopePostureManager, **kwargs):
+        """
+        :param name: the name of the component (usually "Sample Stage")
+        :param role: the role of the component (usually "stage")
+        :param stage_bare: the stage component to be wrapped
+        :param posture_manager: the posture manager to be used for conversion
+        :param **kwargs: additional arguments to be passed to the parent class
+        """
+
+        self._stage_bare = stage_bare
+
+        model.Actuator.__init__(self, name, role, dependencies={"under": stage_bare},
+                                axes=copy.deepcopy(self._stage_bare.axes), **kwargs)
```

Comment:
This makes the sample stage provide 5 axes. I don't think this virtual stage should support rotations (in practice, I don't think the implementation supports any rotation). So it should only provide x, y, and z axes. 
(same for the `.position` and `.speed` VAs)

Reply:
the virtual stage can still rotate and tilt (even as a function of the underlying stage-bare), even if it can't do it itself. if you only have xyz, you can't fully define the position so i don't understand why we wouldn't include it?

Reply:
This actuator is supposed to be in the sample coordinates. So any rotation should happen relative to the standard rotation required for the posture. With the current implementation, that's not the case, the rotation is passed as-is to the stage-bare.  So I'd rather not support it than moving in the wrong way. Also, I don't think anything in the GUI requires rotation in the sample coordinates. Rotation should be need only to change posture (and for the milling angle, but that also should be handled via the Posture Manager/stage-bare). 

So I see it as a "safety feature" to *not* support rotation in the sample stage.

Reply:
ok, removed the rotation axes